src/app/loans/loans-view/undo-disbursal/undo-disbursal.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="container">
```

Comment:
Please move the component within the `loans/loans-view/loan-account-actions` folder.